### PR TITLE
pre commit hook to verify file sizes

### PR DESCRIPTION
**Pull request by seal6363**
Fixes https://github.com/allenai/allennlp/issues/1753

### CODE REVIEW

**Comment by seal6363** [comment]
./scripts/install-hooks.bash

File: scripts/verify.py [hunk 1]
```diff
@@ -47,12 +47,17 @@ def main(checks):
...
+            run("./scripts/check_files_sizes.sh", shell=True, check=True)
+            print("check files' sizes passed")
+
     except CalledProcessError:
         # squelch the exception stacktrace
         sys.exit(1)
 
 if __name__ == "__main__":
-    checks = ['pytest', 'pylint', 'mypy', 'build-docs', 'check-docs', 'check-links', 'check-requirements']
+    checks = ['pytest', 'pylint', 'mypy', 'build-docs', 'check-docs', 'check-links', 'check-requirements', 'check-files-sizes']
```
Comment (schmmd) [inline scripts/verify.py]: Would you rename this "check-large-files"?

File: scripts/verify.py [hunk 2]
```diff
@@ -47,12 +47,17 @@ def main(checks):
             run("./scripts/check_requirements_and_setup.py", shell=True, check=True)
             print("check requirements passed")
 
+        if "check-files-sizes" in checks:
+            print("Checking all added files have size <= 2MB", flush=True)
+            run("./scripts/check_files_sizes.sh", shell=True, check=True)
```
Comment (schmmd) [inline scripts/verify.py]: Do you need to pass a specific size to the script?

Comment (seal6363) [reply scripts/verify.py]: oh yea

**Pull request merged**